src/pages/ActivationPage.tsx

```diff
@@ -63,6 +66,23 @@ const StartPage = (props: RouteComponentProps) => {
     event.target.focus()
   }
 
+  // TODO: Mock this timer
+  /* istanbul ignore next */
```

Comment:
We previously agreed that anytime we were to use `istanbul ignore next` that we would document why and/or where the code is tested. eg: `/* istanbul ignore next: Tested by Cypress */`